ansible_collections/arista/avd/roles/eos_designs/schemas/schema_fragments/wan_virtual_topologies.schema.yml

```diff
@@ -0,0 +1,112 @@
+# Copyright (c) 2023-2024 Arista Networks, Inc.
+# Use of this source code is governed by the Apache License 2.0
+# that can be found in the LICENSE file.
+# yaml-language-server: $schema=../../../../plugins/plugin_utils/schema/avd_meta_schema.json
+# Line above is used by RedHat's YAML Schema vscode extension
+# Use Ctrl + Space to get suggestions for every field. Autocomplete will pop up after typing 2 letters.
+type: dict
+keys:
+  wan_virtual_topologies:
+    type: dict
+    description: |-
+      PREVIEW: WAN Preview
+
+      Configure Virtual Topologies for CV Pathfinder and AutoVPN.
+
+      Auto create a control plane profile/policy/application and enforce it being first in the default VRF.
+    keys:
+      vrfs:
+        type: list
+        description: |-
+          Map a VRF that exists in network_services to an AVT policy.
+          TODO: missing default VRF behavior
+        primary_key: name
+        items:
+          type: dict
+          keys:
+            name:
+              type: str
+              convert_types:
+                - int
+              description: VRF name.
+            policy:
+              type: str
+              description: Name of the AVT policy to apply to this VRF.
+      control_plane_virtual_topology:
+        type: dict
+        description: |-
+          Always injected into the default VRF policy as the first entry.
+
+          By default, if no path-groups are specified, all locally available path-groups
+          are used in the generated load-balance policy.
+        $ref: "eos_designs#/$defs/application_virtual_topologies"
+        keys:
+          name:
+            type: str
+            description: Optional name, if not set `CONTROL-PLANE-PROFILE` is used.
+          id:
```

Comment:
consider removing the id option and just hardcode to 254. Then set max to 253 in the other id fields.